lib/eventstream_rpc_utils.ts

```diff
@@ -0,0 +1,633 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+import * as eventstream_rpc from "./eventstream_rpc";
+import {CrtError, eventstream} from "aws-crt";
+
+/*
+ * Internal utility functions for generated RPC clients to perform normalization, serialization, deserialization, and
+ * validation.
```

Comment:
Just to make sure my understanding is correct: These functions are not necessarily being used right now, but they will be used in the generated RPC clients?

Reply:
Yes